tests/FeedbackTest.php

```diff
@@ -68,7 +74,8 @@ public function testHoneypotAndHoneypot() {
     $this->request
         ->shouldReceive('getQueryParamPOST')
         ->with('user-captcha')
-        ->andReturn(base64_encode(time() - 5 * 60));
+        // 6 seconds ago is more than the default 5 seconds
+        ->andReturn(base64_encode(time() - 6));
```

Comment:
The base64_encode function will do nothing to the value here.

Reply:
I don't understand?
Here the mocked `getQueryParamPOST` method is used to pass parameters, in a roundabout way, to the controller method being tested here. It is given an argument `time() - 6` i.e. an integer representing the Unix timestamp six seconds ago. PHP implicitly converts this to a string. Then base64_encode returns it as a base64 encoded string; and this is the "parameter" that eventually is passed to the controller.

Reply:
Oh, I tested badly, my bad! 😄